### ISSUE TITLE
ImageSelector - Editing an image in an ItemSet opens two copies of the image 

### ISSUE THREAD

**Issue by ljl** [enonic/xp#3780]
Hello.

I have the following contenttype:
```
<?xml version="1.0" encoding="UTF-8"?>
<content-type>
    <display-name>Gallery</display-name>
    <super-type>base:structured</super-type>
    <allow-child-content>true</allow-child-content>
    <form>
        <item-set name="images">
            <label>Image</label>
            <items>
                <input name="image" type="ImageSelector">
                    <label>Image</label>
                    <config>
                        <allowPath>./*</allowPath>
                    </config>
                    <occurrences minimum="0" maximum="1"/>
                </input>
                <input name="label" type="TextLine">
                    <label>Image Text</label>
                    <occurrences minimum="1" maximum="1"/>
                </input>
            </items>
            <occurrences minimum="0" maximum="0"/>
        </item-set>
    </form>
    <x-data mixin="menu-item" />
</content-type>

```

If I select the image, and click edit - i will get two tabs with the same image content.

However, if I dont select the image, i can still click edit and it opens just one copy. Using Enonic 6.6.2.

See attached gif:
![image-edit](https://cloud.githubusercontent.com/assets/414079/19104454/df7fea84-8adc-11e6-8f7e-e669736af2bf.gif)

**Comment by ljl** [comment]
I can also add that the actual edit and remove buttons are not displayed outside of an Item Set unless an image is selected

**Comment by alansemenov** [comment]
Hi Lasse

This bug was fixed in version 6.7, so I suggest you upgrade to the latest version of XP which is 6.7.2
